### DISCUSSION TITLE
鱼塘热榜 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#116] [模板讨论]
鱼塘热榜 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
Failed at 1/4 request,
Fail assert: {"re": "错误", "from": "content"} from failed_asserts,
Request URL: https://api.tophub.fun/ULogin

**Reply by wjf0214** [reply]
这个网站现在应该有问题。等修复看看
